Web/src/components/AddExpenseDialog.jsx

```
@@ -0,0 +1,219 @@
+    };
+
+    const handleSubmit = () => {
+        // Validate form data
+        if ((!formData.isCash && (!formData.date || !formData.description || !formData.category || !formData.account || !formData.amount)) || (formData.isCash && (!formData.date || !formData.description || !formData.category || !formData.amount))) {
+            alert('Please fill in all fields');
+            return;
+        }
+
+        // Convert amount to number and validate
+        const amount = Number(formData.amount);
+        if (isNaN(amount)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)